jujugui/static/gui/src/app/components/profile/credential-list/credential-list.js

```diff
@@ -220,7 +240,7 @@ class ProfileCredentialList extends React.Component {
         controllerIsReady={this.props.controllerIsReady}
         credential={overrides.credential}
         credentials={
-          credentials ? Array.from(credentials).map(credential => credential.name) : []}
+          credentials ? Array.from(credentials).map(credential => credential[0]) : []}
```

Comment:
doesn't this return the tag and not the display name?

Reply:
Yep! That's what we want.